sdk/identity/identity/CHANGELOG.md

```diff
@@ -886,7 +887,7 @@ This release doesn't have the changes from `1.2.4-beta.1`.
 
 ## 1.0.0-preview.1 (2019-06-27)
 
-For release notes and more information please visit https://aka.ms/azsdk/releases/july2019preview
+For release notes and more information please visit <https://aka.ms/azsdk/releases/july2019preview>
```

Comment:
is this change intentional?

Reply:
yes, it's coming from markdownlint https://github.com/DavidAnson/markdownlint/blob/main/doc/md034.md and I figured I can commit it once and not have to think about it anymore 😄